src/sequelize.js

```diff
@@ -818,17 +818,16 @@ class Sequelize {
   async truncate(options) {
     const models = [];
 
-    this.modelManager.forEachModel(
-      model => {
-        if (model) {
-          models.push(model);
-        }
-      },
-      { reverse: false }
-    );
+    this.modelManager.forEachModel(model => {
+      if (model) {
```

Comment:
I know this is also not your fault, but while we're at it, do you know why is this `if` necessary? It seems really odd for `.forEachModel` to return a falsy model :sweat_smile: